hoodie-spark/src/main/scala/com/uber/hoodie/AvroConversionUtils.scala

```diff
@@ -43,6 +48,23 @@ object AvroConversionUtils {
     }
   }
 
+  def createDataFrame(rdd: RDD[GenericRecord], schemaStr: String, ss : SparkSession): Dataset[Row] = {
+    val dateType = convertAvroSchemaToStructType(Schema.parse(schemaStr))
```

Comment:
Is this dateType used anywhere ? I see you re-create this inside the mapPartitions ..

Reply:
Goo catch. I will remove this line